src/Action/Implementation.hs

```diff
@@ -64,6 +71,14 @@ instance HasSendMail ActionExcept ActionEnv Action where
 instance ActionLog Action where
     log msg = actionIO =<< views envLogger ($ msg)
 
+    readEventLog = do
+        cfg <- viewConfig
+        rows :: [EventLogItemCold]
```

Comment:
You mean "event log file not found"?  I can't think of a sane situation in which that happens, but you're probably right that we should catch it.